### PR TITLE
Add `finally`.

### PR DESCRIPTION
**Pull request by epost**

### CODE REVIEW

**Comment by puffnfresh** [comment]
:shipit:

File: src/Control/Monad/Aff.purs [hunk 1]
```diff
@@ -106,6 +106,13 @@ module Control.Monad.Aff
   apathize :: forall e a. Aff e a -> Aff e Unit
   apathize a = const unit <$> attempt a
 
+  -- | Compute `aff1`, followed by `aff2` regardless of whether `aff1` terminated successfully.
+  finally :: forall e a. Aff e a -> Aff e Unit -> Aff e a
```
Comment (jdegoes) [inline src/Control/Monad/Aff.purs]: Can you make this function polymorphic in the 2nd argument?

**Comment by jdegoes** [comment]
Thanks for this! Looks awesome, please just (a) make the function polymorphic in the 2nd arg, and (b) add an example / test to the example file.

**Comment by epost** [comment]
Thanks. :) Voila!

**Comment by jdegoes** [comment]
:+1:

**Pull request merged**